.github/workflows/auto-update-packages.yml

```diff
@@ -0,0 +1,49 @@
+name: Update Packages
+
+on:
+  schedule:
+    - cron: '0 6 * * 3' # 6AM UTC WED => 11PM PDT TUE
+
+jobs:
+  update-packages:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
This is dangerous as this gives the job write access to the repository. And with a 3rd-party action that isn't pinned to a specific commit means the code could change from underneath us and do something malicious.

What specifically requires this permission? Is it the branch creation?

Reply:
This is needed to update contents, this action is writing to the requirements file, and package.json to make upadates.